.github/workflows/examples.yml

```diff
@@ -5,8 +5,6 @@ on:
     branches:
       - main
   pull_request:
-    branches:
-      - main
```

Comment:
This change should be reverted before merging. This change was only included so that we could see the `botan` example run in CI even when the PR was not targetting the `main` branch